cli/src/main/java/com/devonfw/tools/ide/cli/NullIdeRootException.java

```diff
@@ -0,0 +1,15 @@
+package com.devonfw.tools.ide.cli;
+
+/**
+ * {@link CliException} that is thrown if IDE_ROOT == null
+ */
+public class NullIdeRootException extends CliException {
```

Comment:
If you really want to express what you stated in the JavaDoc, it would be better to rename it to `IdeRootIsNullException` so one can better understand and guess what all this is about.
However, an exception that we use to abort without any message should not be named after the condition it was caused by but rather than the problem type it represents and potential handling it may imply.
Therefore I would suggest renaming to `CliCancelException` or `CliExitException`.
Also it would be good if you define a custom exit code like already done in `CliAbortException`.

Reply:
done